src/api/app/controllers/webui/package_controller.rb

```diff
@@ -337,19 +338,23 @@ def rdiff
     end
   end
 
-  def save_new
-    @package = @project.packages.build(name: @package_name)
-    @package.title = params[:title]
-    @package.description = params[:description]
-    if params[:source_protection]
-      @package.flags.build(flag: :sourceaccess, status: :disable)
-    end
-    if params[:disable_publishing]
-      @package.flags.build(flag: :publish, status: :disable)
-    end
+  def new
+    # FIXME: Use the package policy for this
+    authorize @project, :update?
```

Comment:
I use `authorize @project, :update?` since this is what we were using in the project_controller action `new_package`.

Reply:
I will just look into the `sourceaccess` thing.. We have a `source_access?` policy which isn't included in `update?`, but TBH update looks the right one to me too

Reply:
This is temporary, it's going to be the `create?` method which I'm going to create in the package policy in a following PR.